source/developers-guide/shopware-5-plugin-update-guide/index.md

```diff
@@ -110,163 +156,166 @@ The new files contain the following source code:
 {/block}
 ```
 
-#### SwagExample1/Views/emotion/frontend/detail/example1.tpl
-```html
-{block name="frontend_detail_index_detail"}
-    {include file="frontend/swag_example1/detail_extension.tpl"}
-{/block}
-```
+Notice: Template extensions for the responsive template are loaded via the inheritance hierarchy based on the file system. Therefore, this template should be extends via {extends file = ".."}.
 
-#### SwagExample1/Views/responsive/frontend/detail/index.tpl
-```html
-{extends file="parent:frontend/detail/index.tpl"}
+The SwagExample1/Views/responsive/frontend/detail/index.tpl and SwagExample1/Views/emotion/frontend/detail/example1.tpl files serve only as entry points into the original template. The source code for displaying the top seller sliders and the banner element, which was previously located directly in the extended template file, has now been made available globally in a separate template file, and is now simply included by both templates. This has the following advantages:
```

Comment:
fix the pathes and delete the emotion thing

Reply:
D